code/game/jobs/job/command/auxiliary/pilot.dm

```diff
@@ -1,20 +1,20 @@
 /datum/job/command/pilot
-	title = JOB_PILOT
-	total_positions = 2
-	spawn_positions = 2
+	title = JOB_CAS_PILOT
+	total_positions = 1
+	spawn_positions = 1
```

Comment:
```suggestion
	total_positions = 2
	spawn_positions = 2
```
Keep these two. Let there be a co-pilot that can tag team and support the other.